src/main/java/core/basesyntax/service/reports/ReportServiceImp.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.reports;
+
+import java.util.Map;
+
+public class ReportServiceImp implements ReportService {
+    @Override
+    public String create(Map<String, Integer> data) {
```

Comment:
I think we could create some fruitDao field here, and call fruitDao.getData() in this class